client/src/components/HistoryExport/ToRemoteFile.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <div>
+        <b-alert show variant="warning" v-if="errorMessage">
+            {{ errorMessage }}
+        </b-alert>
+        <div v-if="waitingOnJob">
+            <loading-span message="Executing history export job, this will likely take a while." />
+        </div>
+        <div v-else-if="jobComplete">
+            <b-alert show variant="success" dismissible @dismissed="reset">
+                <h4>Done!</h4>
+                <p>History appears to have been successfully exported.</p>
```

Comment:
_appears_ does not breed with confidence from the user's perspective. As a user, I'd like a _yes/no_ outcome, not a _maybe_ outcome...

Reply:
I mean... have you ever gotten it to work 😆. When does this work? Who is confident in history imports and exports?

I'll change the language though

Reply:
Changed.